input/vulnerabilities/Samsung_WifiHs20UtilityService.json

```diff
@@ -22,5 +22,11 @@
   , "projectzeroblog-huntinggalaxy" : {
       "url" : "https://googleprojectzero.blogspot.co.uk/2015/11/hack-galaxy-hunting-bugs-in-samsung.html"
     }
-  }
+  },
+  "Surface": ["remote", "local", "webview", "filesystem"],
```

Comment:
Can a webview cause such a download by default?

Reply:
Yes, according to https://bugs.chromium.org/p/project-zero/issues/detail?id=489&redir=1